metadata-ingestion/src/datahub/ingestion/source/tableau/tableau_common.py

```diff
@@ -979,7 +979,6 @@ def get_filter_pages(query_filter: dict, page_size: int) -> List[dict]:
         len(query_filter.keys()) == 1
         and query_filter.get(c.ID_WITH_IN)
         and isinstance(query_filter[c.ID_WITH_IN], list)
-        and len(query_filter[c.ID_WITH_IN]) > 100 * page_size
```

Comment:
do you have a sense for why this code was added in the first place? should it be outright removed, or simply tweaked

I don't like removing code unless I understand why it was added in the first place

Reply:
I am not sure, @treff7es / @mayurinehate might be aware.

Reply:
This was introduced [here](https://github.com/datahub-project/datahub/pull/10295) . Added PR description to help understand why .

I see some the original logic changed in [this](https://github.com/datahub-project/datahub/pull/11230)  which may have caused `page_size` to break.

Reply:
I confirm this change is safe to get in. It would have no adverse effects.